modules/oxide/package.json

```diff
@@ -32,7 +32,7 @@
     "README.md",
     "LICENSE.txt"
   ],
-  "dependencies": {
+  "devDependencies": {
```

Comment:
This was incorrect and breaks the published `@tinymce/oxide` skin for 6.4.0

We will be publishing a 6.4.1 in the next week or two to fix this.